game/hud/src/components/HUD/index.tsx

```diff
@@ -7,7 +7,6 @@
 import * as React from 'react';
 import { connect } from 'react-redux';
 import { client, ql, events } from 'camelot-unchained';
-import { useConfig } from 'camelot-unchained/lib/graphql/react';
```

Comment:
Why is this removed?

Reply:
not sure there, looking at git history, it was added by you, then removed during a merge. added back.

Reply:
OH I know what removed it, since it was unused it was auto removed do to some optimize settings I have.